src/shared/util.js

```diff
@@ -174,22 +174,28 @@ export const hyphenate = cached((str: string): string => {
 })
 
 /**
- * Simple bind, faster than native
+ * Bind function context
  */
-export function bind (fn: Function, ctx: Object): Function {
+function ownBind (fn: Function, ctx: Object): Function {
   function boundFn (a) {
-    const l: number = arguments.length
+    const l = arguments.length
     return l
       ? l > 1
         ? fn.apply(ctx, arguments)
         : fn.call(ctx, a)
       : fn.call(ctx)
   }
-  // record original fn length
+
   boundFn._length = fn.length
   return boundFn
 }
 
+function nativeBind(fn: Function, ctx: Object): Function {
+  return fn.bind(ctx)
```

Comment:
It's missing the optional arguments, cause different behavior to ownBind

Reply:
I see no optional arguments in `ownBind`. Are you about `fn._length`? If yes. It shouldn't be a problem because we should check both `length` and `_length` since we don't know exactly was the `fn` bound via our `ownBind` or not. e.g.: https://github.com/vuejs/vue/blob/7cf188e134fe7bfc9e8a16b763fb85b18eff1eac/src/platforms/weex/runtime/modules/transition.js#L73